<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosImportFromTextFile</class>
 <widget class="QWidget" name="ReosImportFromTextFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import From Text File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFile">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Delimiters:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDelimiters"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonAddDelimiter">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/moveLeft.svg</normaloff>:/images/moveLeft.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRemoveDelimiter">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/moveRight.svg</normaloff>:/images/moveRight.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDelimiter">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>space</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>,</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>;</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Header line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxHeaderLine">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>First data line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxFirstLine">
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
